tests/TestCase.php

```diff
@@ -42,6 +44,30 @@ public function getFixturePath(string $fixtureName): string
     }
 
     protected function getEnvironmentSetUp($app): void
+    {
+        $this->includeEnv();
+
+        $this->setupDb($app);
+    }
+
+    protected function getPackageProviders($app): array
+    {
+        return [
+            MediaServiceProvider::class,
+        ];
+    }
+
+    protected function includeEnv(): void
+    {
+        $pathToEnv = str_replace('/vendor/orchestra/testbench-core/laravel', '',  base_path());
+        $name = env('APP_ENV') ? '.env.' . env('APP_ENV') : null;
+        if (File::exists("{$pathToEnv}/{$name}")) {
+            $dotenv = Dotenv::createImmutable($pathToEnv, $name);
+            $dotenv->load();
+        }
```

Comment:
could we use something like it:

```suggestion
        Dotenv::createImmutable(__DIR__ . '/..', '.env.testing')->load();
```

Reply:
Yes. I fixed it.